bazel/cc_configure.bzl

```diff
@@ -108,7 +59,6 @@ cc_autoconf = repository_rule(
         "USE_CLANG_CL",
         "CC",
         "CFLAGS",
-        "CXX",
```

Comment:
When building with GCC, does the final link happen with `gcc` or `g++`?

Reply:
`gcc`